utils/iio_rwdev.c

```diff
@@ -46,10 +46,16 @@ static const struct option options[] = {
 
 static const char *options_descriptions[] = {
 	"[-t <trigger>] [-b <buffer-size>]"
+#if WITH_LOCAL_DMABUF_API == 1
+    "[-C] "
+#endif
```

Comment:
It makes more sense to eliminate the `if` directives from both iio_rwdev.c and iio_stresstest.c. This PR only addresses the local context, but at some point, these changes need to be extended for the other contexts (network, usb, serial, xml). And for that uses cases, when running iio_rwdev on a system where local context is not available, then WITH_LOCAL_DMABUF_API won't be defined (it's not applicable there) but it could be defined on the target device where iiod is running.
Therefore, it's best to keep the cmdline argument all the time and users will receive an error message in the scenario where CMA is not available.